packages/web-app-external-sites/src/App.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="external-site-container">
```

Comment:
This should be a `<main>` tag for a11y reasons... we have a button to jump to the `main` content. Editor apps have that from `AppWrapper.vue` but standalone apps need to define it.